drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/projects_importer.rb

```diff
@@ -120,9 +158,10 @@ def upsert_orgs
     def upsert_projects
       file = 'Project.csv'
 
+      hud_columns = GrdaWarehouse::Hud::Project.hmis_configuration(version: '2024').keys.map(&:to_s)
```

Comment:
Should this be `2022`? The old expected columns included `TrackingMethod`...

Reply:
Thanks for noticing that. This method is expected to run after the 2022->2024 transform is completed. My understanding was that we don't want to carry forward the 2022 fields here. Perhaps there's some convention on what to do with deprecated fields. Should we continue to update them?

Reply:
No we don't need to keep updating the 2022 fields. I think this is correct. At this point, the file should be in 2024 format.